Misc/NEWS.d/next/Library/2024-12-01-16-43-38.gh-issue-127478.fHI_y9.rst

```diff
@@ -0,0 +1 @@
+Enhancement to support FTP connection with NAT scenario.
```

Comment:
I'll try to come up with a better NEWS entry here for tomorrow but otherwise thank you.